build.gradle

```diff
@@ -459,60 +435,37 @@ dependencies {
         exclude group: "net.bytebuddy", module: "byte-buddy"
         exclude group: "net.bytebuddy", module: "byte-buddy-agent"
     }
-    testImplementation ('com.github.hmcts:rd-commons-lib:v0.0.14'){
-        exclude group: 'org.springframework.boot', module: 'spring-boot-starter-web'
-        exclude group: 'org.apache.groovy', module: 'groovy-json'
-        exclude group: 'org.apache.groovy', module: 'groovy'
-        exclude group: 'org.apache.groovy', module: 'groovy-xml'
-    }
-    testImplementation group: 'org.springframework', name: 'spring-test'
-
-    testImplementation group: 'net.bytebuddy', name: 'byte-buddy', version: versions.bytebuddy
-    testImplementation group: 'net.bytebuddy', name: 'byte-buddy-agent', version: versions.bytebuddy
```

Comment:
removal of libs is causing 1.14.19 to be pulled in transitively, but we declare in 'versions' to use 1.15.3. please restore

Reply:
retracting comment